modulefiles/module_base.noaacloud.lua

```diff
@@ -5,8 +5,11 @@ Load environment to run GFS on noaacloud
 local spack_mod_path=(os.getenv("spack_mod_path") or "None")
 prepend_path("MODULEPATH", spack_mod_path)
 
+load("gnu")
```

Comment:
Is there bootstrapping or something going on here?

Reply:
That's not the case on any of the on-prem machines, which is why I asked.

Reply:
I'm guessing this has to do with the way Intel was built on the cloud.  The spack recipe requires gnu in order to get a valid glibc library to build against.  But the spack-written module files don't usually include it because they invariably use the system glibc library.  *shrug*